src/ansys/fluent/core/launcher/standalone_launcher.py

```diff
@@ -261,7 +261,11 @@ def __call__(self):
                 if self.meshing_mode:
                     session.tui.file.read_case(self.case_file_name)
                 else:
-                    session.read_case(self.case_file_name, self.lightweight_mode)
+                    session.file.read(
+                        file_type="case",
+                        file_name=self.case_file_name,
+                        lightweight_setup=self.lightweight_mode,
+                    )
```

Comment:
In my understanding, the previous version wasn't set up correctly. 

We can still support the PyFluent-handled "lightweight mode" (which seems to not be quite the same as the Fluent settings-only "lightweight setup" mode), I believe these changes have an example of how it could be set up so it works correctly: https://github.com/ansys/pyfluent/pull/2268/files

Reply:
https://github.com/ansys/pyfluent/pull/2252/files#r1424056929

Reply:
@seanpearsonuk Thanks for clarifying. It seems like PyFluent "lightweight mode" (loading the mesh in the background) should be called `lightweight_setup` and Fluent's settings-only mode (not loading the mesh at all, previously referred to as "lightweight setup") should actually be called `settings_only` and they're getting confused with each other.

Perhaps the Fluent launcher and read methods, as well as their documentation, should explicitly distinguish between a `lightweight_setup` mode (load the mesh in the background) and a `settings_only` mode (settings only, do **not** load the mesh in the background). It should emulate the Fluent GUI:

![image](https://github.com/ansys/pyfluent/assets/144711134/028d5efc-60e2-4557-bd13-91600dc39fc6)

What would be the best path forward? I wonder if it would be best to condense all the `solver.read_*()` and `solver.file.read_*()` methods into just `solver.file.read()` to ensure that there's only one way to read files (in the spirit of [PEP 20](https://peps.python.org/pep-0020/)).

Reply:
@ansjsia I would say that in the short term, to fix the `TypeError` issue when specifying `case_file_name` you correctly identified here, we should make the changes as outlined in the links Sean and I shared above (both links have the same solution). 

Also, can you please add a test covering the `case_file_name` argument so we make sure we don't break this again in the future? 

Then, after discussion, and in a separate PR, we will hopefully be able to properly address the "lightweight/lite mode/setup settings-only" naming confusion (which I expect will require more extensive changes).